tests/phpunit/tests/block-bindings/postMetaSource.php

```diff
@@ -0,0 +1,263 @@
+<?php
+/**
+ * Tests for Block Bindings API "core/post-meta" source.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.5.0
+ *
+ * @group blocks
+ * @group block-bindings
+ */
+class Tests_Block_Bindings_Post_Meta_Source extends WP_UnitTestCase {
+	protected static $post;
+	protected static $wp_meta_keys_saved;
+
+	/**
+	 * Modify the post content.
+	 *
+	 * @param string $content The new content.
+	 */
+	private function getModifiedPostContent( $content ) {
```

Comment:
WordPress does not use camelCase. Please use snake_case everywhere (except for `wpSetUpBeforeClass` and `wpTearDownAfterClass`, these are special fixture methods). Consult the [coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/) for more details.

Reply:
My bad, I assumed it worked that way only for tests. Changed in [this commit](https://github.com/WordPress/wordpress-develop/pull/6197/commits/4cdf2419df23567913499d0d8dd82216dde49df8).